hub/auto/unstructured/image_classification.py

```diff
@@ -150,15 +150,7 @@ def structure(  # type: ignore
 
         with ds, iterator:
             for file_path in iterator:
-                try:
-                    image = hub.read(file_path)
-                except PIL.Image.UnidentifiedImageError:
-                    skipped_files.append(file_path.name)
-                    iterator.set_description(
-                        'Ingesting "%s" (%i files skipped)'
-                        % (self.source.name, len(skipped_files))
-                    )
-                    continue
+                image = hub.read(file_path)
```

Comment:
why?

Reply:
@farizrahman4u and I discussed this in the #develop channel on slack. This is bad code that wasn't needed. At no point does hub.read give PIL.Image.UnidentifiedImageError.

Reply:
maybe you can except the CorruptedSampleError

Reply:
The problem with CorruptedSampleError only appears when we use the object, which has been taken care of here along with other possible exceptions (line 171) ->
![image](https://user-images.githubusercontent.com/61134339/166663558-e9bd2546-7fd0-491e-97ee-e96b9444f1df.png)

hub.read returns a "sample" object which when being initialized does not open the file so I don't think we should have problems with corrupted data when initializing. It feels like we only need to take care of errors such CorruptedSampleError and SampleDecompressionError when it is being used for example as above.